### DISCUSSION TITLE
How do I start the eleventy server on a new tab automatically

### DISCUSSION THREAD

**Discussion by Rajdeepc** [11ty/eleventy#2914] [Q&A]
Doing ```eleventy --serve``` It starts the server but doesn't open the webpage automatically. How can i do this?

**Comment by synj00** [comment]
It seems that it is not tied into the underlying os system files that would open the browser / create tab in open browser. I had this problem after installing vs-code. A reboot fixed it. Did you also try npx @11ty/eleventy --serve

**Comment by pdehaan** [comment]
Add `open https://localhost:8081` (or whatever port you're using) to some terminal command?
Although I imagine that'd open a new tab each time and not reuse the existing tab, so it could spawn a small army of tabs.
